ci.py

```diff
@@ -28,35 +28,35 @@ class Builder():
     rootdir = check_output(["git", "rev-parse", "--show-toplevel"]).decode('utf-8').strip()
 
     def __init__(self):
-        self.custom_config = self.read_custom_config()
+        self.config_override = self.read_config_override()
         self.config = self.read_config()
 
         self.merge_configs()
 
-
-    def read_custom_config(self):
-        with open(YAML_CONFIG_CUSTOM, 'r') as f:
+    def read_config_override(self):
+        with open(YAML_CONFIG_OVERRIDE, 'r') as f:
             return yaml.load(f, Loader=yaml.FullLoader)
 
     def read_config(self):
         with open(YAML_CONFIG, 'r') as f:
             return yaml.load(f, Loader=yaml.FullLoader)
 
     def write_custom_config(self, config):
-        with open(YAML_CONFIG_CUSTOM, 'w') as f:
+        with open(YAML_CONFIG_OVERRIDE, 'w') as f:
             yaml.dump(config, f, sort_keys=False)
 
     def merge_configs(self):
-        if self.custom_config is not None:
-            # Yep we have high complexity here but list is short
```

Comment:
I think this comment make no sense, creating map could not be beneficial as well. It needs to be tested.
Plus, it is Python. It needs to be clear, not fast ;)